graveyard.json

```diff
@@ -19,8 +27,8 @@
     "dateClose": "2015-12-21",
     "dateOpen": "2005-09-14",
     "description": "Google Blog Search API was a way to search blogs utilizing Google.",
-    "link": "https://en.wikipedia.org/wiki/Google_Blog_Search",
-    "name": "Google Blog Search API",
+    "link": "https://revolv.com/",
```

Comment:
Looks like you edited in the wrong place.

Reply:
Oops.  Sorry about that.